content/ctl_ohai.md

```diff
@@ -24,6 +24,31 @@ aliases = ["/ctl_ohai.html"]
 
 The following examples show how to use the Ohai command-line tool:
 
+### Query for a specific attribute
+
+To query for an attribute with the ohai command-line tool, pass the attribute
+as an argument
+
+```bash
+ohai os
+```
+
+Which would fetch the value found within Chef in `node[:os]` and return 
```

Comment:
```suggestion
Which would fetch the value found within Chef in `node['os']` and return 
```

Reply:
Good idea, that's better language

Reply:
Since I merged this GitHub Suggested change from @tas50 the commit wasn't "Signed-Off by" and so the DCO CI check is now failing. Do I need to do anything to address that?